New Unity Project/Assets/PickUpLoader.cs

```diff
@@ -1,18 +1,16 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-public class PickUpLoader : MonoBehaviour {
+﻿using UnityEngine;
 
+public class PickUpLoader : MonoBehaviour
+{
     public string[] _prefabNames = new string[] { "DummyPickup", "InvertControls", "LowerSpeed", "SpeedUp" };
-    public int count = 5;
-    public Vector3 pickupScale = new Vector3(0.5f, 0.5f, 0.5f);
+    public int _count = 5;
```

Comment:
Shouldn't public members be PascalCase after C# code style conventions?

Reply:
You're right about that. It wasn't my intention to use a public field. Maybe I was a little bit to lazy that evening.

Reply:
I will fix it tomorrow